src/components/review-list.jsx

```diff
@@ -0,0 +1,32 @@
+import "./review-board.css";
+import { useEffect } from "react";
+import { getReviewBoard } from "../api";
+
+function ReviewList({ reviews, setReviews }) {
+  useEffect(() => {
+    getReviewBoard().then((data) => {
+      setReviews(data);
+    });
+  }, []);
+  return (
+    <ul className="review-list">
+      {reviews.map((review) => {
```

Comment:
Could you give the user some feedback that the page is loading if the fetch request takes longer than expected?